plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/OTelTracingTelemetry.java

```diff
@@ -32,15 +32,15 @@ public class OTelTracingTelemetry implements TracingTelemetry {
      */
     public OTelTracingTelemetry(OpenTelemetry openTelemetry) {
```

Comment:
We could pass `OpenTelemetrySdk` directly by changing `OTelResourceProvider` to return `OpenTelemetrySdk` (we create it anyway like that)

Reply:
Agreed, Already made that change :)